packages/quantic/force-app/main/default/lwc/quanticFacet/__tests__/quanticFacet.test.js

```diff
@@ -231,9 +333,674 @@ describe('c-quantic-facet', () => {
+            selectors.clearSelectionButton
+          );
+
+          await clearSelectionButton.click();
+
+          expect(functionsMocks.deselectAll).toHaveBeenCalledTimes(1);
+          expect(functionsMocks.updateText).toHaveBeenCalledTimes(1);
+          expect(functionsMocks.updateText).toHaveBeenCalledWith('');
+        });
+      });
+
+      describe('when the facet is not collapsed', () => {
+        it('should display the facet body', async () => {
+          const element = createTestComponent();
+          await flushPromises();
+
+          const facetBody = element.shadowRoot.querySelector(
+            selectors.facetBody
+          );
+          expect(facetBody).not.toBeNull();
+        });
+
+        it('should correctly display the facet collapse toggle facet', async () => {
+          const expectedIcon = 'utility:dash';
+          const expectedCSSClass = 'facet__collapse';
+          const element = createTestComponent();
+          await flushPromises();
+
+          const facetCollapseToggle = element.shadowRoot.querySelector(
+            selectors.facetCollapseToggle
+          );
+          expect(facetCollapseToggle).not.toBeNull();
+          expect(facetCollapseToggle.iconName).toBe(expectedIcon);
+          expect(facetCollapseToggle.classList.contains(expectedCSSClass)).toBe(
+            true
+          );
+        });
+      });
+
+      describe('when the facet is collapsed', () => {
+        it('should not display the facet body', async () => {
+          const element = createTestComponent({
+            ...defaultOptions,
+            isCollapsed: true,
+          });
+          await flushPromises();
+
+          const facetBody = element.shadowRoot.querySelector(
+            selectors.facetBody
+          );
+          expect(facetBody).toBeNull();
+        });
+
+        it('should correctly display the facet collapse toggle facet', async () => {
+          const expectedIcon = 'utility:add';
+          const expectedCSSClass = 'facet__expand';
+
+          const element = createTestComponent({
+            ...defaultOptions,
+            isCollapsed: true,
+          });
+          await flushPromises();
+
+          const facetCollapseToggle = element.shadowRoot.querySelector(
+            selectors.facetCollapseToggle
+          );
+          expect(facetCollapseToggle).not.toBeNull();
+          expect(facetCollapseToggle.iconName).toBe(expectedIcon);
+          expect(facetCollapseToggle.classList.contains(expectedCSSClass)).toBe(
+            true
+          );
+        });
+
+        describe('when a facet value is selected', () => {
+          beforeEach(() => {
+            facetState = {
+              ...initialFacetState,
+              values: [{value: 'example value', numberOfResults: 10}],
+              hasActiveValues: true,
+            };
+          });
+
+          it('should display the clear selection button', async () => {
```

Comment:
this is pretty much the same as your test for: `when the facet has active values` is there much value in a seperate test for when the isCollapsed property is true?

Reply:
Yes there is a value, we wanna confirm that the clear selection button is displayed even when the facet is collapsed and that it's not among the parts that were collapsed and hidden.